pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -220,6 +162,115 @@ def run_func(self, *args, **kwargs):
 
         return decorator
 
+    def _get_pattern_lst(self, source_path, timestamps=False):
+        '''
+        return an order list sort by file name pattern
+        '''
+        if timestamps:
+            # [world_timestamps.npy, world_001_timestamps.npy, world_002_timestamps.npy]
+            pattern, _ = os.path.splitext(source_path)
+            suffix = "_timestamps.npy"
+            tmp = sorted(glob.glob(pattern + '*' + suffix))
+            return tmp[-1:] + tmp[:-1]
```

Comment:
Please add a comment to facilitate the understanding of what this line does and why it is required.